pkgs/applications/version-management/cvsq/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl, makeWrapper,
+  cvs, perl, nettools, findutils,
+  rsync, coreutils, diffutils } :
```

Comment:
Personally I prefer every argument in it's own line, no big deal though but I think it makes sense and the diffs nicer.